### PR TITLE
Add benchmark for combining signatures

### PR DESCRIPTION
**Pull request by fhaynes**
This benchmark is based on how the `test_threshold_sig` uses the combine_signatures. Let me know if this benchmarks it correctly. 

Is it worth adding in testing different thresholds?

### CODE REVIEW

**Comment by afck** [comment]
Looks good, thank you! But let's test different thresholds, yes; maybe just the same ones as the degrees for the arithmetic operations.

**Comment by fhaynes** [comment]
@afck Looks like this is failing due to the clippy changes in beta. I found this comment in the source:
```
    # Note: `beta` should be removed along with `RUST_NEXT` after the 1.28
    #       stable release on 2018-09-13.
```
It looks like the beta channel is still in use, though. Making the changes to get it to work with beta break it on stable. Should we just remove beta from the travis build process?

**Comment by afck** [comment]
You're right, that looks like the whole `RUST_NEXT` thing and `rustup toolchain install` should be removed. (@mbr: Please correct me if I'm wrong.)

**Comment by afck** [comment]
…also the symlinking and the `+${RUST_NEXT}`—I imagine that with clippy-preview in stable now, everything should just work without these?

**Comment by afck** [comment]
I'll try to fix it: #36

File: benches/bench.rs [hunk 1]
```diff
@@ -86,4 +87,41 @@ mod poly_benches {
...
+            move |b, &&threshold| {
+                let sk_set = SecretKeySet::random(threshold, &mut rng);
+                let pk_set = sk_set.public_keys();
+                let sigs: BTreeMap<_, _> = [5, 8, 7, 10]
+                    .iter()
+                    .map(|&i| {
+                        let sig = sk_set.secret_key_share(i).sign(msg);
+                        (i, sig)
+                    }).collect();
+                b.iter(|| pk_set.combine_signatures(&sigs))
```
[the remark this answers is not in the document]
Comment (fhaynes) [reply benches/bench.rs]: Correct you are! I pushed an update that takes it into account. This made the benchmark, um, considerably slower. =)

**Comment by afck** [comment]
I rebased it and opened #39 instead.

**Pull request closed without merging**